airflow/cli/cli_config.py

```diff
@@ -551,6 +551,9 @@ def string_lower_type(val):
     action="store_true",
 )
 ARG_RAW = Arg(("-r", "--raw"), argparse.SUPPRESS, "store_true")
+ARG_CARRIER = Arg(
+    ("-c", "--carrier"), help="Context Carrier, containing the injected context for the task span", nargs="?"
```

Comment:
```suggestion
    ("-c", "--carrier"), help="Context Carrier, containing the injected context for the OTel task span", nargs="?"
```

And shouldn't this be `nargs=1`?

Reply:
If otel isn't configured, then `Trace` points to `EmptyTrace` and `Trace.inject()` returns `{}`. 

Now, that I took another look at it, this is still interpreted as a string value and not None.

I'll make the changes.

Reply:
`nargs=1` is converting the value into a list which complicates handling the retrieved value. We can remove `nargs` entirely.